expression/builtin_math_vec.go

```diff
@@ -177,24 +177,6 @@ func (b *builtinCosSig) vectorized() bool {
 	return true
 }
 
-func (b *builtinTanSig) vecEvalReal(input *chunk.Chunk, result *chunk.Column) error {
```

Comment:
@b41sh Why is it removed?
It seems that there is some wrong result when resolving conflicts 🤔

Reply:
There is someting wrong when merge, I will recovery it